causalpy/pymc_models.py

```diff
@@ -73,6 +161,33 @@ def score(self, X, y) -> pd.Series:
             The Bayesian :math:`R^2` is not the same as the traditional coefficient of
             determination, https://en.wikipedia.org/wiki/Coefficient_of_determination.
 
+        Example
+        --------
+        >>> import causalpy as cp
+        >>> import numpy as np
+        >>> import pymc as pm
+        >>> from causalpy.pymc_models import ModelBuilder
+        >>> class MyToyModel(ModelBuilder):
+        ...     def build_model(self, X, y, coords):
+        ...         with self:
+        ...             X_ = pm.MutableData(name="X", value=X)
+        ...             y_ = pm.MutableData(name="y", value=y)
+        ...             beta = pm.Normal("beta", mu=0, sigma=1, shape=X_.shape[1])
+        ...             sigma = pm.HalfNormal("sigma", sigma=1)
+        ...             mu = pm.Deterministic("mu", pm.math.dot(X_, beta))
+        ...             pm.Normal("y_hat", mu=mu, sigma=sigma, observed=y_)
+        >>> rng = np.random.default_rng(seed=42)
+        >>> X = rng.normal(loc=0, scale=1, size=(200, 2))
+        >>> y = rng.normal(loc=0, scale=1, size=(200,))
+        >>> model = MyToyModel(
+        ...         sample_kwargs={"chains": 2, "draws": 2000, "progressbar": False}
+        ... )
+        >>> model.fit(X, y)
+        Inference...
+        >>> round(model.score(X, y),2) # using round() to simplify doctest
```

Comment:
The results should be more stable, and the doctest less fussy if you pass in random_seed=rng (I believe) in the sample_kwargs dict

Reply:
Thanks, with the seed and the `+NUMBER` doctest option the `round` call is no longer needed.